v21.2/cost-based-optimizer.md

```diff
@@ -64,29 +67,26 @@ If you need to turn off automatic statistics collection, follow the steps below:
     > SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;
```

Comment:
To speed up build times, we've been replacing all `{% include copy-clipboard.html %}` tags with `{% include_cached copy-clipboard.html %}`. So I'm just noting them in this review.